app/services/open_library_service/fetch.rb

```diff
@@ -0,0 +1,14 @@
+module OpenLibraryService
+  class Fetch
+    class_attribute :http_client
+    include HTTParty
+    base_uri 'https://openlibrary.org/api'
+    def self.book_info(isbn)
+      @options = { query: { format: 'json', jscmd: 'data' } }
+      @options[:query][:bibkeys] = "ISBN:#{isbn}"
+      response = get('/books', @options)
+      return response if response.code == 200
+      raise OpenLibraryServiceErrors::BookInfoFetchError, response: {}
+    end
```

Comment:
It's kinda hard on the eyes when everything is crammed up and there are no blank spaces separating functions